### PR TITLE
add jpy build

### PR DESCRIPTION
**Pull request by IslamAlam**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/jpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "jpy" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/bcdev/jpy.git
```
Comment (devinrsmith) [inline recipes/jpy/meta.yaml:16]: I think this should be updated to https://github.com/jpy-consortium/jpy

Comment (IslamAlam) [reply recipes/jpy/meta.yaml]: @devinrsmith thanks so much for noticing it. I have updated it.

**Comment by IslamAlam** [comment]
@conda-forge/help-python again ready for review,

File: recipes/jpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/jpy-consortium/jpy.git 
+    git_rev: {{ version }}
+    git_depth: -1 # (Defaults to -1/not shallow)
+    folder: jpy
```
Comment (carterbox) [inline recipes/jpy/meta.yaml:19]: Please use release tars instead of checking out the repository.

File: recipes/jpy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (carterbox) [inline recipes/jpy/meta.yaml:7]: Please clean up all of the example comments.

File: recipes/jpy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}            # [unix]
+    - {{ compiler('fortran') }}        # [unix]
+    - {{ compiler('c') }}              # [unix]
```
Comment (carterbox) [inline recipes/jpy/meta.yaml:44]: Why are compilers only needed for unix and not for Windows? If you are not planning to build for Windows, then there should be a skip statement in the build section.

File: recipes/jpy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+
+  # For python packages, it is useful to run pip check. However, sometimes the
+  # metadata used by pip is out of date. Thus this section is optional if it is
+  # failing.
+
+  commands:
```
Comment (carterbox) [inline recipes/jpy/meta.yaml:68]: Every package needs tests. Probably for this package you should try to import jpy in both python and java.

**Review by carterbox — CHANGES_REQUESTED** [review]

**Pull request closed without merging**